docs/get_float.adoc

```diff
@@ -11,27 +11,44 @@ get_float - prompts user for a line of text from stdin and returns the equivalen
 
 *#include <cs50.h>*
 
-*float get_float(string prompt);*
+*float get_float(const string format, ...);*
```

Comment:
Sounds reasonable to me. Any preference on what the prototypes in `cs50.h` should be?